pennylane_lightning/lightning_qubit/lightning_qubit.toml

```diff
@@ -1,129 +1,112 @@
-schema = 1
+schema = 2
 
-[device]
-name = "lightning.qubit"
+[operators.gates.native]
 
-[operators]
-# Observables supported by the device
-observables = [
-        "PauliX",
-        "PauliY",
-        "PauliZ",
-        "Hadamard",
-        "Hermitian",
-        "Identity",
-        "Projector",
-        "SparseHamiltonian",
-        "Hamiltonian",
-        "Sum",
-        "SProd",
-        "Prod",
-        "Exp",
-        "LinearCombination",
-]
+CNOT                   = { properties = [ "invertible",                 "differentiable" ] }
+ControlledPhaseShift   = { properties = [ "invertible",                 "differentiable" ] }
+CPhase                 = { properties = [ "invertible",                 "differentiable" ] }
```

Comment:
Ali is creating a PR that removes `CPhase` from the device list.
```suggestion
```

Reply:
@maliasadi can you link your pr here?

Reply:
CPhase is removed